src/ralph/models/xapi/fields/objects.py

```diff
@@ -70,3 +71,31 @@ class PageObjectField(ObjectField):
     """
 
     definition: PageObjectDefinitionField
+
+
+class VideoObjectDefinitionField(BaseModelWithConfig):
+    """Represents the `object.definition` xAPI field for played xAPI statement.
+
+    WARNING: It doesn't include the recommended `description` field.
+
+    Attributes:
+       type (str): Consists of the value `https://w3id.org/xapi/video/activity-type/video`.
+       name (str): Consists of the name of the video being played.
+       description (str): Consists of a description of the video being played.
+    """
+
+    type: ACTIVITY_TYPE_VIDEO = ACTIVITY_TYPE_VIDEO.__args__[0]
+    description: str
+    name: str
```

Comment:
I think the `name` and `description` are LanguageMaps ? Like `{"en": "some string"}`
Or does marsha fills the `name`/`description` field with a string ?

Reply:
Those are language maps, you are right. :1st_place_medal: